test/sdk/restapi/alpha/swagger/api/swagger.yaml

```diff
@@ -0,0 +1,203 @@
+---
```

Comment:
All this generated code will need a Apache header, and a note about it being generated.

There are some boilerplate files here: https://github.com/googleforgames/agones/tree/main/build for both go and yaml that we use for headers for generated code. Figured they might be useful.

Can't see if the swagger command line tool will do this for you, if not, you can steal from: https://github.com/googleforgames/agones/blob/main/build/build-sdk-images/go/gen.sh

Reply:
Added Apache header to generated files.